### PR TITLE
Mac os

### PR DESCRIPTION
**Pull request by sedwards**
c2d0acc is simply an attempt to make the log window more aesthetically pleasing by providing a border on the frame.

718e2a6 is more of a pain as without this patch, my sftp connections only work one out of a few dozen times. I've not been able to track down what is causing the buggy behavior on macOS, but I am assuming it is a platform bug as it is not 100% consistent.

### CODE REVIEW

File: lib/sockutils.c [hunk 1]
```diff
@@ -345,7 +345,12 @@ gftp_fd_set_sockblocking (gftp_request * request, int fd, int non_blocking)
   else
     flags &= ~O_NONBLOCK;
 
+#ifdef __APPLE__
+  int opt = 1;
```
Comment (masneyb) [inline lib/sockutils.c]: This will only turn on non-blocking sockets but it will not turn it back off. This should be: `int opt = non_blocking;` since the non-blocking mode is switched in the code base.

Does this work for you on OS/X? I don't have access to that platform so I can't test this functionality there. O_NONBLOCK is defined by POSIX and is the preferred way of doing that.

More of this code can be wrapped in an #ifdef block. Look further up inside that same function. The existing socket flags and fetched, modified, and then the socket is set to non-blocking. So you should be able to do:

$ifdef __APPLE__
     int opt = non_blocking;
     if (ioctl(fd, FIONBIO, &opt) < 0) 
#else
     // Most of the existing function...

File: src/gtk/gftp-gtk.c [hunk 2]
```diff
@@ -1072,6 +1072,9 @@ CreateFTPWindows (GtkWidget * ui)
   gtk_paned_pack2 (GTK_PANED (logpane), log_table, 1, 1);
   gtk_box_pack_start (GTK_BOX (mainvbox), logpane, TRUE, TRUE, 0);
 
+  gtk_container_set_border_width (GTK_CONTAINER (transfer_scroll), 5);
+  gtk_container_set_border_width (GTK_CONTAINER (tempwid), 5);
```
Comment (masneyb) [inline src/gtk/gftp-gtk.c]: This should be in a separate pull request. I don't have any issues with this and will merge it separately.

File: lib/sockutils.c (hunk 1 above)
Comment (sedwards) [inline lib/sockutils.c]: Thanks I will look at this

File: src/gtk/gftp-gtk.c (hunk 2 above)
Comment (sedwards) [inline src/gtk/gftp-gtk.c]: I appreciate it. Sorry will make sure to do separate patches/pull requests for each feature to keep things clearly separated.
